src/qt/peertablemodel.cpp

```diff
@@ -15,29 +15,29 @@
 #include <QList>
 #include <QTimer>
 
-bool NodeLessThan::operator()(const CNodeCombinedStats &left, const CNodeCombinedStats &right) const
+bool NodeLessThan::operator()(const CNodeCombinedStats& left, const CNodeCombinedStats& right) const
 {
-    const CNodeStats *pLeft = &(left.nodeStats);
-    const CNodeStats *pRight = &(right.nodeStats);
+    const CNodeStats* pLeft = &(left.nodeStats);
+    const CNodeStats* pRight = &(right.nodeStats);
 
-    if (order == Qt::DescendingOrder)
+    if (order == Qt::DescendingOrder) {
         std::swap(pLeft, pRight);
+    }
 
-    switch(column)
-    {
+    switch (column) {
     case PeerTableModel::NetNodeId:
         return pLeft->nodeid < pRight->nodeid;
     case PeerTableModel::Address:
         return pLeft->addrName.compare(pRight->addrName) < 0;
-    case PeerTableModel::Subversion:
-        return pLeft->cleanSubVer.compare(pRight->cleanSubVer) < 0;
     case PeerTableModel::Ping:
         return pLeft->m_min_ping_usec < pRight->m_min_ping_usec;
     case PeerTableModel::Sent:
         return pLeft->nSendBytes < pRight->nSendBytes;
     case PeerTableModel::Received:
         return pLeft->nRecvBytes < pRight->nRecvBytes;
-    }
+    case PeerTableModel::Subversion:
+        return pLeft->cleanSubVer.compare(pRight->cleanSubVer) < 0;
+    } // no default case, so the compiler can warn about missing cases
```

Comment:
I think this comment is not true, as the `switch` parameter is an `enum`, not a `enum class`.

Reply:
You're right--thanks!